ui/components/MeshModelRegistry/MesheryTreeView.js

```diff
@@ -176,70 +196,87 @@ const MesheryTreeViewItem = ({
+                <StyledTreeItem
+                  nodeId={`${registrantID ? `${registrantID}.1.` : ''}${modelDef.id}.${
+                    selectedVersionedModel.id
+                  }.1`}
+                  data-id={`${registrantID ? `${registrantID}.1.` : ''}${modelDef.id}.${
+                    selectedVersionedModel.id
+                  }.1`}
+                  labelText={`Components (${
+                    selectedVersionedModel.components ? selectedVersionedModel.components.length : 0
+                  })`}
+                >
+                  {selectedVersionedModel.components &&
+                    selectedVersionedModel.components.map((component, subIndex) => {
+                      // console.log("component", component);
+                      return (
+                        <StyledTreeItem
+                          key={subIndex}
+                          nodeId={`${registrantID ? `${registrantID}.1.` : ''}${modelDef.id}.${
+                            selectedVersionedModel.id
+                          }.1.${component.id}`}
+                          data-id={`${registrantID ? `${registrantID}.1.` : ''}${modelDef.id}.${
+                            selectedVersionedModel.id
+                          }.1.${component.id}`}
+                          labelText={component.displayName}
+                          onClick={() => {
+                            setShowDetailsData({
+                              type: COMPONENTS,
+                              data: component,
+                            });
+                          }}
+                        />
+                      );
+                    })}
+                </StyledTreeItem>
+                <StyledTreeItem
+                  nodeId={`${registrantID ? `${registrantID}.1.` : ''}${modelDef.id}.${
+                    selectedVersionedModel.id
+                  }.2`}
+                  data-id={`${registrantID ? `${registrantID}.1.` : ''}${modelDef.id}.${
+                    selectedVersionedModel.id
+                  }.2`}
+                  labelText={`Relationships (${
+                    selectedVersionedModel.relationships
+                      ? selectedVersionedModel.relationships.length
+                      : 0
+                  })`}
+                >
+                  <RelationshipTree
+                    handleToggle={handleToggle}
+                    handleSelect={handleSelect}
+                    expanded={expanded}
+                    selected={selected}
+                    data={selectedVersionedModel.relationships}
+                    view={MODELS}
+                    setShowDetailsData={setShowDetailsData}
+                    idForKindAsProp={`${registrantID ? `${registrantID}.1.` : ''}${modelDef.id}.${
+                      selectedVersionedModel.id
+                    }.2`}
+                  />
+                </StyledTreeItem>
+              </>
+            ) : (
+              <CircularProgress />
```

Comment:
Okay you already added here thanks, please add this prop to make sure it gets update with theme, also please test after update
```suggestion
              <CircularProgress color="inherit"/>
```